chia/wallet/wallet.py

```diff
@@ -526,3 +526,11 @@ async def create_spend_bundle_relative_chia(self, chia_amount: int, exclude: Lis
             self.wallet_state_manager.constants.MAX_BLOCK_COST_CLVM,
         )
         return spend_bundle
+
+    async def get_coins_to_offer(self, asset_id: Optional[bytes32], amount: uint64) -> Set[Coin]:
+        if asset_id is not None:
+            raise ValueError(f"The standard wallet cannot offer coins with asset id {asset_id}")
+        balance = await self.get_confirmed_balance()
+        if balance < amount:
+            raise Exception(f"insufficient funds in wallet {self.id()}")
+        return await self.select_coins(amount)
```

Comment:
same here, this can raise if there is not enough spendable funds

Reply:
ditto to above